androidApp/src/main/java/com/pulselink/data/sms/SmsCloudSynchronizer.kt

```
@@ -0,0 +1,314 @@
+                subscriptionTier,
+                syncedThreads,
+                syncedMessages,
+                hasReadSms,
+                "ok"
+            )
+
+            if (targetedThreadId <= 0 && deviceContactsRepository.hasContactsPermission() && settings.remoteWebAccessEnabled) {
+                syncDeviceContacts(user.uid)
+            }
+            return Result.success(Unit)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)